ext/ckeditor4/templates/CRM/Ckeditor4/Form/CKEditorConfig.tpl

```diff
@@ -47,7 +47,7 @@
 {/literal}</style>
 {* Force the custom config file to reload by appending a new query string *}
 <script type="text/javascript">
-  {if $configUrl}CKEDITOR.config.customConfig = '{$configUrl}?{php}print str_replace(array(' ', '.'), array('', '='), microtime());{/php}'{/if};
+  {if $configUrl}CKEDITOR.config.customConfig = '{$configUrl}?{$smarty.now}'{/if};
```

Comment:
did you test this one? it's kinda hard to  be sure if it is right

Reply:
I tested it in that it successfully sticks a timestamp in there as the query string, which should force it to reload per the comment. I'm not entirely sure what it is specifically supposed to change though, so I didn't test beyond that.